XRTK-Core/Packages/com.xrtk.core/Definitions/Devices/MixedRealityInteractionMapping.cs

```diff
@@ -200,6 +170,15 @@ public MixedRealityInteractionMapping(MixedRealityInteractionMapping mixedRealit
         /// </summary>
         public string Description => description;
 
+        [SerializeField]
+        [Tooltip("Influences how the Interaction determines state changes that will raise the InputAction.")]
+        private StateChangeType stateChangeType;
+
+        /// <summary>
+        /// Influences how the Interaction determines state changes that will raise the <see cref="MixedRealityInputAction"/>.
+        /// </summary>
+        public StateChangeType StateChangeType => stateChangeType;
```

Comment:
might just comment this out until it's actually implemented.

I'm not entirely sure how this can be used/implemented yet.